tests/mongo.yml

```diff
@@ -249,3 +253,24 @@ results:
       - mongo/jstests/aggregation/use_query_sort.js
       # fails when updating the maximum size of the capped collection
       - mongo/jstests/core/capped/capped_resize.js
+      # SyntaxError: import declarations may only appear at top level of a module
+      - mongo/jstests/aggregation/accumulators/min_n_max_n.js
+      - mongo/jstests/aggregation/bugs/cond.js
+      - mongo/jstests/aggregation/bugs/groupMissing.js
+      - mongo/jstests/aggregation/bugs/match.js
+      - mongo/jstests/aggregation/expressions/filter.js
+      - mongo/jstests/aggregation/expressions/set.js
+      - mongo/jstests/aggregation/ifnull.js
+      - mongo/jstests/aggregation/sources/collStats/query_exec_stats.js
+      - mongo/jstests/aggregation/testall.js
+      - mongo/jstests/core/query/awaitdata_getmore_cmd.js
+      - mongo/jstests/core/query/cursor/getmore_invalidated_cursors.js
+      - mongo/jstests/core/query/cursor/tailable_cursor_invalidation.js
+      - mongo/jstests/core/query/cursor/tailable_getmore_batch_size.js
+      - mongo/jstests/core/query/elemmatch/elemmatch_object.js
+      - ongo/jstests/core/query/elemmatch/elemmatch_value.js
+      - mongo/jstests/core/write/crud_api.js
+      - mongo/jstests/readonly/get_more.js
+      - mongo/jstests/core/query/elemmatch/elemmatch_value.js
```

Comment:
The increase in fails could be because we likely need to bump the shell and server to v7.

For example, the [elemmatch_value.js](https://github.com/FerretDB/dance/actions/runs/7102197681/job/19332010404#step:8:135) test does not error when I run it with shell version v7.1.0-alpha-1852-g22a5016.

```sh
byron@byron-ThinkCentre-M910q:~/workdir/dance/tests/mongo$ mongo jstests/core/query/elemmatch/elemmatch_value.js
MongoDB shell version v7.1.0-alpha-1852-g22a5016
connecting to: mongodb://127.0.0.1:27017/?compressors=disabled&gssapiServiceName=mongodb
Implicit session: session { "id" : UUID("5d40559f-9095-477c-9238-badf33711428") }
MongoDB server version: 6.0.8
WARNING: shell and server versions do not match
```